scaffold/cmd/source.go

```diff
@@ -57,44 +44,47 @@ type scaffoldData struct {
 }
 
 func runScaffoldSource(org string, name string, outputDir string) error {
-	tpl, err := template.New("source").ParseFS(sourceFS,
-		"templates/source/.github/workflows/*.tpl",
-		"templates/source/*.tpl",
-		"templates/source/plugin/*.tpl",
-		"templates/source/client/*.tpl",
-		"templates/source/resources/*.tpl",
-	)
-	if err != nil {
-		return fmt.Errorf("failed to parse templates: %w", err)
-	}
-
 	data := scaffoldData{
 		Org:  org,
 		Name: name,
 	}
-	for templatePath, filePath := range scaffoldTemplates {
```

Comment:
Before we had an hardcoded map of template names filenames to output files. This didn't allow different templates with the same base name of a file (e.g. `client/client.go.tpl` and `plugin/client.go.tpl`)